src/modules/recipes/RecipeItem/index.jsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import CardActionArea from "@mui/material/CardActionArea";
+import CardActions from "@mui/material/CardActions";
+import { useNavigate } from "react-router-dom";
+
+export const RecipeItem = ({ recipe }) => {
+  const navigate = useNavigate();
+
+  const handleRecipeClick = () => {
+    console.log("Recipe Clicked", recipe);
+    navigate(`/recipes/${recipe.idMeal}`);
```

Comment:
Make sure to update the Recipe Details page with MUI as well, it seems unstyled right now.